src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/InputInterpreter.cs

```diff
@@ -215,10 +215,22 @@ public static void GallonHandler(ref string[] split, CultureInfo culture)
             }
         }
 
+        /// <summary>
+        /// Add a "0" if the first query character is "."
+        /// </summary>
+        public static void PrefixZero(ref string[] split)
+        {
+            if (split[0][0] == '.')
```

Comment:
I disagree.
In some cultures people use comma instead of period as a decimal separator.
I think it would be better to check `CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator` value in runtime and use `CultureInfo.InvariantCulture` is nests.

Reply:
I did not know that! I've updated the codes as suggested. Thank you.